test/factory/users.js

```diff
@@ -7,8 +7,7 @@ factory.define('User', db.User, {
   id: factory.sequence('user.id', n => n),
   firstName: faker.name.firstName(),
   lastName: faker.name.lastName(),
-  email: `${faker.internet.userName().toLocaleLowerCase()}@wolox.com`,
-  // password: 'contrasena1234',
+  email: factory.sequence('user.email', n => `${faker.internet.userName().toLowerCase()}${n}@wolox.com`),
```

Comment:
What is this sequence for?

Reply:
faker used the same email to create several users in one test and using the sequence always generates a different one